graphql_compiler/schema/schema_info.py

```diff
@@ -84,6 +84,11 @@
     # a sqlalchemy table. Column types that do not exist for this dialect are not allowed.
     'tables',
 
+    # dict mapping every graphql object type or interface type name in the schema to
+    # a string name of its primary key column. The column cannot contain nulls and must
+    # be unique.
+    'primary_keys',
```

Comment:
How much work would it be to design this to allow composite (multi-column) primary keys from day 1? Maybe if we have `Dict[str, Collection[str]]` as the type?

Reply:
I wouldn't do it in this PR, since it's already day 2. If we support multi-column primary keys, then the `join_descriptors` should also assume multi-column foreign keys.